src/ui/setup/index.ts

```diff
@@ -34,9 +40,26 @@ export function bootstrapApp() {
   const store = bootstrapStore();
   window.store = store;
 
-  registerStoreObserver(store, updatePrefs);
+  // Check if we have any preferences saved for this replay and apply
+  // them to the store. Todo: Do this step before we create the store.
+  asyncStore.replaySessions
+    .then((sessions: ReplaySessions) => {
+      const session = sessions[getRecordingId()!];
+
+      console.log({ session, session });
 
-  setupAppHelper(store);
+      if (!session) {
+        return;
+      }
+
+      store.dispatch(setViewMode(session.viewMode));
+      store.dispatch(setShowVideoPanel(session.showVideoPanel));
+      store.dispatch(setShowEditor(session.showEditor));
```

Comment:
What do you think about moving this into `src/ui/setup/dynamic/devtools.ts`, kind of similar to what I did over at: https://github.com/RecordReplay/devtools/pull/3899/files#diff-54af09a258c2f6698b5fd88700d5ce17b3c7c77ffc5e2431420d5c9e39532942R77-R79?

Reply:
Also, not necessary for this PR, but it seems like it could be worth it to have a `setReplaySession` action which can update all of these at the same time for when we switch recordings.